config/flags.go

```diff
@@ -149,6 +149,8 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.Uint(LogRotaterMaxAgeKey, 0, "The maximum number of days to retain old log files based on the timestamp encoded in their filename. 0 means retain all old log files.")
 	fs.Bool(LogRotaterCompressEnabledKey, false, "Enables the compression of rotated log files through gzip.")
 	fs.Bool(LogDisableDisplayPluginLogsKey, false, "Disables displaying plugin logs in stdout.")
+	fs.Bool(LogAutoAmplificationKey, false, "Amplifies log level to DEBUG when health checks fail, reverts back once they succeed again or duration defined by --log-auto-amplification-max-duration elapses.")
```

Comment:
I agree that this should not enabled by default in application but should be enabled with a default infra config.

Reply:
I don't think what you're talking about is in this repository, but I could be wrong

Reply:
Yes, I was just asserting the `false` default choice sounds better to me.